<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>50</y>
     <width>191</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>MFC</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="mf_n2_groupBox">
      <property name="title">
       <string>MFC N2</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Setpoint</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="mfc_n2_setpoint_doubleSpinBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Flow</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="mfc_n2_flow_doubleSpinBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="mfc_ar_groupBox">
      <property name="title">
       <string>MFC Ar</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Setpoint</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="mfc_ar_setpoint_doubleSpinBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Flow</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="mfc_ar_flow_doubleSpinBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="mfc_h2_groupBox">
      <property name="title">
       <string>MFC H2</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Setpoint</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="mfc_h2_setpoint_doubleSpinBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Flow</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="mfc_h2_flow_doubleSpinBox"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="pc_groupBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>110</y>
     <width>171</width>
     <height>129</height>
    </rect>
   </property>
   <property name="title">
    <string>Pressure Controller</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,1">
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Setpoint</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="pc_setpoint_doubleSpinBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Pressure</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="pc_pressure_doubleSpinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Temperature</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Pyrometer Process Controller</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_6" columnstretch="0,1">
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Setpoint</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="pyro_sv_doubleSpinBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Actual</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="pyro_pv_doubleSpinBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="pyro_outp_doubleSpinBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Outside Temp  Process Controller</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_8" columnstretch="0,0">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="outside_pv_doubleSpinBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Actual</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QCheckBox" name="run_checkBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Live Update</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Pyrometer Temp</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="pyro_temp_doubleSpinBox">
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
